pymoto/modules/assembly.py

```diff
@@ -235,38 +235,76 @@ class AssembleMass(AssembleGeneral):
 
     Args:
         domain: The domain to assemble for -- this determines the element size and dimensionality
+        ndof: Amount of dofs per node
+            Mass, Damping: ndof = domain.dim
+            Else: ndof = 1
         *args: Other arguments are passed to AssembleGeneral
 
     Keyword Args:
-        rho: Base density
+        material_property: Material property to use in the element matrix
+            Mass: Density (rho)
+            Damping: Damping parameter
+            Thermal capacity: Thermal capacity * Density
         bcdiagval: The value to put on the diagonal in case of boundary conditions (bc)
         **kwargs : Other keyword-arguments are passed to AssembleGeneral
     """
 
-    def _prepare(self, domain: DomainDefinition, *args, rho: float = 1.0, bcdiagval=0.0, **kwargs):
-        # Element mass matrix
-        # 1/36 Mass of one element
-        mel = rho * np.prod(domain.element_size)
+    def _prepare(self, domain: DomainDefinition, *args, material_property: float = 1.0, ndof: int = 1, bcdiagval=0.0, **kwargs):
```

Comment:
Is there a reason that 'ndof' is a separate input, while it can also be obtained from 'domain' (saving users one argument to specify)? If not, use the 'ndof' property from 'domain'

Reply:
This is required to distinguish between a mechanical mass matrix (3dofs) and a thermal mass matrix (1dof)